Kickstarter-iOS/Views/Controllers/CuratedProjectsViewController.swift

```diff
@@ -0,0 +1,138 @@
+import Library
+import Prelude
+import UIKit
+
+final class CuratedProjectsViewController: UIViewController {
+  // MARK: - Properties
+
+  private lazy var collectionView: UICollectionView = {
```

Comment:
Curious about why you chose to make this a collection view? The plan for this screen is either to embed `DiscoveryPageViewController` underneath the header view, or to start from scratch to display the `DiscoveryPostcardCell`s which are `UITableViewCell`s. I think if we go from scratch a table view would still make the most sense imo?